Dockerfile

```diff
@@ -1,13 +1,31 @@
-FROM golang:1.11 as builder
+FROM golang:1.12 AS builder
 
-COPY . /go/src/github.com/Lookyan/netramesh
-WORKDIR /go/src/github.com/Lookyan/netramesh
-RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-extldflags "-static"' -o main ./cmd/main.go
+WORKDIR /src
 
+ADD . .
+
+ENV GOOS        linux
+ENV GOARCH      amd64
+ENV CGO_ENABLED 0
+
+RUN go build  -o /go/bin/netramesh \
+              -mod vendor \
+              -a -installsuffix cgo \
+              -ldflags '-extldflags "-static"' \
+              ./cmd/main.go
+
+
+FROM alpine:latest AS service
+
+LABEL maintainers="Alexander Lukyanchenko <digwnews@gmail.com>, \
+Mikhail Leonov <lm@kodix.ru>, \
+Kamil Samigullin <kamil@samigullin.info>"
+
+RUN adduser -D -H -u 1000 service
```

Comment:
Is it possible to install deps using `apk` with this user?

Reply:
do you mean inside runnable container or at build step?

Reply:
inside. To debug issues without `docker exec -u root` from host.

Reply:
Nope, maybe it will be better to build separated image to do that. I can remove this section but what about security? Is it ok to run the service as root by default?